templates/includes/sidebar.html

```diff
@@ -0,0 +1,13 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="#">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>

```

Comment:
The `href` attribute currently uses a placeholder `#`. Replace this with the actual URL path for the Cars page to make the link functional.